frameworks/Scala/play2-scala/play2-scala/build.sbt

```diff
@@ -1,11 +1,13 @@
 name := "play2-scala"
 
-version := "2.6.0"
+version := "1.0-SNAPSHOT"
 
-scalaVersion := "2.11.11"
+scalaVersion := "2.12.3"
 
-val root = (project in file(".")).enablePlugins(PlayScala)
+val root =
+  (project in file(".")).
+  enablePlugins(PlayScala, PlayNettyServer).
+  disablePlugins(PlayAkkaHttpServer)
```

Comment:
@godenji Why did you disable akka-http and instead used the legacy netty backend?

Reply:
@mkurz for this round, performance. Maybe things have changed in latest version of Play, but for a while Akka was lagging well behind Netty.

Reply:
Hmm.. All the other play2-scala and play2-java benchmarks use akka-http as well.. And it's the default now in Play... So wouldn't it be a better and more realistic "real-world" benchmark if we would use akka-http instead? Did you run netty vs akka-http benchmarks for that particular project on your machine?

Reply:
I have a pull request in line to upgrade to latest Play version and change also change that project to akka-http: https://github.com/TechEmpower/FrameworkBenchmarks/compare/master...mkurz:play2-scala-fixes?expand=1

Reply:
I don't have a strong preference, feel free to switch it over to Akka (though the last round's results for play2-scala were [excellent with Netty](https://www.techempower.com/benchmarks/#section=data-r14&hw=ph&test=json)).

Reply:
I switched it to Akka (https://github.com/TechEmpower/FrameworkBenchmarks/pull/3065), because all other projects are on akka as well. However I think about to just create benchmarks with netty for *each* of the projects. This can be done in the setup bash scripts, e.g. by just copying each project folder and replace the `build.sbt` with one containing the netty setting enabled.

Reply:
Maybe we should defer to play core devs on this. I like your idea of having netty and akka benchmarks, though it may be that Lightbend wants to highlight Akka rather than a 3rd party library that seems to be on the way out of Play; in which case we can just go with your latest PR.

Again, I don't have a vested interest either way, would just like to see Play perform as well or better than the last round.

Reply:
Hi, I'm a Lightbend dev so I'll chime in on this issue. The plan is to eventually move to only using an Akka HTTP backend. Play 2.6 (and maybe 2.7) is a transitional release where we support both Netty and Akka HTTP backends. If you want to write tests to run both backends then that will be useful information for us, because we'll be able to see if they're equivalent. However, if there's only enough energy to maintain one backend then Akka HTTP would be the better choice. So if you want to do both backends then that would be great. :)

Not sure if you're doing this, but with Netty you can also enable the native transport on Linux:
* https://www.playframework.com/documentation/2.6.x/SettingsNetty
* http://netty.io/wiki/native-transports.html

Reply:
@richdougherty Since #3065 was just merged, all the benchmark tests are now on akka-http. However I will provide a netty test for each benchmark as well, that shouldn't be difficult since we don't need to copy - and therefore maintain - two almost identical projects for each benchmark. The Techempower benchmark suite allows you to set up bash scripts for each test so we can just copy the akka-http test for each run and just replace the `build.sbt` with one that configures netty.
Also I am aware of the native transport setting :wink:

Reply:
@mkurz you're also going to want to update [applicable lines](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Scala/play2-scala/play2-scala/conf/application.conf#L46-L59) in play2-scala application.conf, which have native `transport = "native"`, etc. Netty conf settings already in place.

@richdougherty having akka-http and netty tests for the upcoming round will be useful data indeed. Given how well Netty performed in the last round I'll be impressed if Akka can hang with it in the next round -- game on ;-)

Reply:
Yes looking forward to seeing the results!